plugins/parcel-transformer-markdown-html/lib/index.js

```diff
@@ -68,6 +68,21 @@ module.exports = new Transformer({
 					: Promise.resolve(null)
 			]);
 
+			let tokens;
+			// we've specified a tokens file to load from @atlas-tokens
+			if (attributes.token) {
+				try {
+					const jsm = require(`@microsoft/atlas-tokens/dist/${attributes.token}.js`);
```

Comment:
I think when the main is updated you'll be able to use `const tokens = require('@microsoft/atlas-tokens')`